package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "todo-list1",
+  "version": "1.0.0",
+  "description": "> Todo list is a web app help you to management well your important tasks",
+  "main": "webpack.config.js",
+  "scripts": {
+    "start": "webpack",
+    "serve": "webpack serve --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Alfredbis29/Leaderboard.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Alfredbis29/Leaderboard/issues"
+  },
+  "homepage": "https://github.com/Alfredbis29/Leaderboard#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^6.1.11",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^4.9.2"
+  },
+  "dependencies": {
+    "css-loader": "^6.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "uuid": "^8.3.2",
+    "webpack-dev-server": "^4.7.4"
+  }
+}
```

Comment:
- [ ] Please install webpack, webpack-cli, and webpack-dev-server. These packages should be included in your package.json file and should be added as devDependencies too so Kindly fix that.

Reply:
_ooooh i did mistake_